test/alternative_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "spec_helper.rb"
+require "update-alternatives/model/alternative"
+
+describe UpdateAlternatives::Alternative do
+
+  describe ".all" do
+    subject { UpdateAlternatives::Alternative.all }
```

Comment:
This is a nitpicking comment, so don't take it too seriously. In the past I used to abuse of explicit `subject` (and I probably did it in the tutorial). But in this case I'm not sure if it's a good idea to use `subject`.

In any case, when using it it's better to add a more explicit label like this.
```ruby
subject(:all_alternatives) { described_class.all }
```

Then your code will look like
```ruby
expect(all_alternatives).to all(be_a(UpdateAlternatives::Alternative))
```
which looks more self explanatory than `subject`